
Exception: java.lang.VerifyError thrown from the UncaughtExceptionHandler in thread "main"
